src/Application.php

```diff
@@ -55,6 +57,10 @@ protected function getDefaultCommands()
     {
         $commands = parent::getDefaultCommands();
 
+        // Create dependencies
+        $githubApi = new GitHubApi();
```

Comment:
If we are concerned about API limits we should employ a caching layer as cached API requests don't impact on the API rate limit

Reply:
I'd also add a GitHubApi getter onto the commands that need it rather than instantiating it with the application

Reply:
Yeah? I'd thought I'd go with dependency injection here. Happy to change though

Reply:
Well, DI would be cool, but can we do it lazily?

Reply:
This particular dependency doesn't do anything until you call `$githubApi->getClient()`, so I guess it's kind of lazy :D